src/dev/getting_started.md

```diff
@@ -83,6 +83,65 @@ required) for this directory instead of the default Rust compiler.
 Then you can build and test the `yk` repo using `cargo xtask` commands, for
 example `cargo xtask test`.
 
+## Rust Analyzer
+
+[Rust-analyzer](https://github.com/rust-analyzer/rust-analyzer) (RA hencforth)
```

Comment:
s/hencforth/henceforth/

Unfortunately "RA" isn't a common acronym for rust-analyzer: I suggest that it would be better to spell it out in full each time.

Reply:
That's why we define it in brackets!

But if you insist :)

Reply:
I don't think the acronym is useful enough here to make the overhead of asking the reader to remember it worthwhile.